compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/SymbolFactory.java

```diff
@@ -375,8 +393,18 @@ public BallerinaMemberTypeSymbol createTupleMember(BVarSymbol symbol) {
         return new BallerinaMemberTypeSymbol(context, symbol, type);
     }
 
+    /**
+     * Create a named path segment symbol.
+     *
+     * @param symbol {@link BResourcePathSegmentSymbol} to convert
+     * @return {@link BallerinaNamedPathSegment} generated
+     */
+    public BallerinaNamedPathSegment createResourceNamedPathSegment(BResourcePathSegmentSymbol symbol) {
```

Comment:
Is this method names consistent with others? Including vs not including resource.

Reply:
Good catch. changed.